### PR TITLE
DOC: Use x1 and x2 in the heaviside docstring.

### PR DESCRIPTION
**Pull request by WarrenWeckesser**
Also remove a duplicate $PARAMS.

Closes gh-9314.

### CODE REVIEW

File: numpy/core/code_generators/ufunc_docstrings.py [hunk 1]
```diff
@@ -928,25 +928,24 @@ def add_newdoc(place, name, doc):
 
     The Heaviside step function is defined as::
 
-                             0   if x < 0
-        heaviside(x, h0) =  h0   if x == 0
-                             1   if x > 0
```
Comment (eric-wieser) [inline numpy/core/code_generators/ufunc_docstrings.py]: I might be inclined to leave this section as is, as the intent is far clearer. The arguments are positional anyway, so it's not incorrect.,.

Comment (WarrenWeckesser) [inline numpy/core/code_generators/ufunc_docstrings.py]: Are there other numpy ufuncs that do that?  If not, I'd rather stick to numpy's documentation patterns and not introduce new names in the docstring.

**Comment by eric-wieser** [comment]
Unrelatedly - should this target 1.13.1? In general, it'd be nicer if documentation fixes could make it out sooner than releases

File: numpy/core/code_generators/ufunc_docstrings.py (hunk 1 above)
Comment (adeak) [inline numpy/core/code_generators/ufunc_docstrings.py]: As far as I can tell only `heaviside` does this, the only other irregularity being `y1`/`y2` in [the docs of `modf`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.modf.html) (every other instance calls these `out1`/`out2`.

I believe I found the source of the double `$PARAMS$`: [`isnat` has none](https://github.com/numpy/numpy/blob/master/numpy/core/code_generators/ufunc_docstrings.py#L1723-L1725). Is this minor enough, or should I open a new issue with this?

Comment (eric-wieser) [inline numpy/core/code_generators/ufunc_docstrings.py]: The source of this double $PARAMS$ is a careless find-and-replace by me. I think the cause of the missing one is the addition of `isnat` being merged after #9026, without the PR being updated

Comment (adeak) [inline numpy/core/code_generators/ufunc_docstrings.py]: I decided to try and fix the missing $PARAMS of `isnat` in [#9323](https://github.com/numpy/numpy/pull/9323)

**Comment by charris** [comment]
@eric-wieser Don't usually backport documentation bug/improvements, but the case of new functions is a bit special.

File: numpy/core/code_generators/ufunc_docstrings.py (hunk 1 above)
Comment (charris) [inline numpy/core/code_generators/ufunc_docstrings.py]: I prefer the original also, but that can wait until the generation of the signature is improved so we can make it agree with the rest of the docstring.

**Pull request merged**